castle/configuration.py

```diff
@@ -25,19 +25,31 @@
 # 500 milliseconds
 REQUEST_TIMEOUT = 500
 FAILOVER_STRATEGIES = ['allow', 'deny', 'challenge', 'throw']
+HOST = 'api.castle.io'
+PORT = 443
+URL_PREFIX = '/v1'
```

Comment:
maybe `ENDPOINT_VERSION`?

Reply:
I would like to stay with the same name across sdks.